web/src/lib/utils/analytics.ts

```diff
@@ -1732,8 +1730,6 @@ export default {
             event: "graduation_phase_interactions",
             legacy_event_action: "click",
             legacy_event_category: "tax_registration_nudge_button",
-            legacy_event_label: "show_tax_registration_modal",
-            action: "show_tax_registration_modal",
             item: "tax_registration_nudge_button",
           });
         },
```

Comment:
[Boulder] Remove lines `1728-1741`. We're not calling `tax_registration_nudge_button` any more. The only instance where this was firing was in `SidebarCardTaxRegisterdForTaxesNudge`, but you removed that.